frontend/src/component/admin/billing/BillingDashboard/BillingPlan/BillingPlan.tsx

```diff
@@ -104,52 +104,40 @@ export const BillingPlan: FC<IBillingPlanProps> = ({ instanceStatus }) => {
 
     const planPrice = price[instanceStatus.plan];
     const seats = instanceStatus.seats ?? 5;
-    const [freeAssigned, setFreeAssigned] = useState(0);
-    const [paidAssigned, setPaidAssigned] = useState(0);
-    const [paidAssignedPrice, setPaidAssignedPrice] = useState(0);
-    const [finalPrice, setFinalPrice] = useState(planPrice);
-    const [totalCost, setTotalCost] = useState(0);
-    const includedTraffic = isPro() ? proPlanIncludedRequests : 0;
-    const [overageCost, setOverageCost] = useState(0);
-    const traffic = useInstanceTrafficMetrics(currentPeriod.key);
-
-    useEffect(() => {
-        setFreeAssigned(Math.min(eligibleUsers.length, seats));
-    }, [users]);
 
-    useEffect(() => {
-        setPaidAssigned(eligibleUsers.length - freeAssigned);
-    }, [freeAssigned]);
+    const freeAssigned = Math.min(eligibleUsers.length, seats);
+    const paidAssigned = eligibleUsers.length - freeAssigned;
+    const paidAssignedPrice = price.user * paidAssigned;
 
-    useEffect(() => {
-        setPaidAssignedPrice(price.user * paidAssigned);
-    }, [paidAssigned]);
-
-    useEffect(() => {
-        setFinalPrice(planPrice + paidAssignedPrice);
-    }, [paidAssignedPrice]);
-
-    const flagEnabled = useUiFlag('displayTrafficDataUsage');
+    const displayTrafficDataUsageEnabled = useUiFlag('displayTrafficDataUsage');
```

Comment:
might have been using the wrong flag all along for this - should be estimateTrafficDataUsageEnabled